mkinitcpio

```diff
@@ -423,6 +425,11 @@ if [[ -t 1 ]] && (( _optcolor )); then
     try_enable_color
 fi
 
+if [[ -n $_d_flag_hooks && -n $_d_flag_install ]]; then
```

Comment:
Shouldn't this be either or? Why not allow mix and match of hooks and install scripts?

Reply:
Needs a second flag and split up the logic. Don't see a usecase currently

Reply:
Yes, to split this, we would need a install hooks dir flag and a hooks dir flag. As I've mentioned before, since they are tightly coupled, there's no need for separating this. Unless we start allowing run hooks that do not have a install counterpart, I don't see a reason for splitting. Also, I don't see this flag being very useful outside of testing.